### ISSUE TITLE
a problem downloading from snapshot build

### ISSUE THREAD

**Issue by virgonia** [FirebirdSQL/firebird#7226]
a problem occurred downloading from a build snapshot.
The problem has been happening since 1-2 days ago.

http://web.firebirdsql.org/download/snapshot_builds/win/5.0

![image](https://user-images.githubusercontent.com/30924531/177060043-25cc1dd4-e4a7-4bb7-97d0-d057dec377d8.png)

**Comment by mrotteveel** [comment]
Yes, there is a problem with the server. I expect it will get fixed in the next few days. In the meantime, you can use the builds generated by the GitHub Actions as a workaround:

See, https://github.com/FirebirdSQL/firebird/actions, for Firebird 5.0, pick the "increment build number" from master, this is equivalent to the snapshots.

**Comment by virgonia** [comment]
many thanks for response
